src/pydata_sphinx_theme/__init__.py

```diff
@@ -201,6 +201,20 @@ def update_templates(app, pagename, templatename, context, doctree):
         app.add_js_file(None, body=js)
 
 
+def add_inline_math(node):
+    """The math output are ignored by sphinx in the output generated by the theme.
+    This method returns the html translation of a math node to hydrate the html
+    output generated by this file.
+
+    related to the behaviour of a normal math node from:
+    https://github.com/sphinx-doc/sphinx/blob/master/sphinx/ext/mathjax.py#L28
```

Comment:
```suggestion
    """Render a node with HTML tags that activate MathJax processing.
    This is meant for use with rendering section titles with math in them, because
    math outputs are ignored by sphinx in headers.

    related to the behaviour of a normal math node from:
    https://github.com/sphinx-doc/sphinx/blob/master/sphinx/ext/mathjax.py#L28
```

Reply:
it only applies to the header navbar because it's the only place we are not relying on the generated Toctree object. as we are writting down the HTML manually (without using the docutils nodes) the math one get ignored at the `astext()` call